src/test/ui/enum/union-in-enum.stderr

```diff
@@ -0,0 +1,40 @@
+warning: variant `union` should have an upper camel case name
```

Comment:
Could you `#![allow(..)]` these warnings in the test? They aren't relevant to what this is testing.